lib/cli/options.ts

```diff
@@ -42,11 +41,11 @@ export class Option {
     if (value && typeof value === 'string') {
       return '' + value;
     }
-    return null;
+    return '';
```

Comment:
> Returning '' will probably cause an error in TypeScript.
 
So this was the "before":

```typescript
  getString(): string {
    let value = this.getValue_();
    if (value && typeof value === 'string') {
      return '' + value;
    }
    return null;
  }
```

and this is the "after":

```typescript
  getString(): string {
    let value = this.getValue_();
    if (value && typeof value === 'string') {
      return '' + value;
    }
    return '';
  }
```

Given that the return type is `string`, it seems `null` should have caused TS to complain here.